arbor/cable_cell_param.cpp

```diff
@@ -139,4 +139,35 @@ locset cv_policy_fixed_per_branch::cv_boundary_points(const cable_cell& cell) co
     return points;
 }
 
+locset cv_policy_every_sample::cv_boundary_points(const cable_cell& cell) const {
+    const unsigned nbranch = cell.morphology().num_branches();
+    if (!nbranch) return ls::nil();
+
+    // Ignore interior_forks flag, but if single_root_cv is set, take sample indices only from branches 1+.
+    // Always include branch proximal points, so that forks are trivial.
+
+    std::vector<locset> points;
+
+    if (flags_&cv_policy_flag::single_root_cv) {
+        points.push_back(mlocation{0, 0.});
+        points.push_back(mlocation{0, 1.});
+        for (unsigned i = 1; i<nbranch; ++i) {
+            points.push_back(mlocation{i, 0.});
+            for (msize_t sample_idx: util::make_range(cell.morphology().branch_indexes(i))) {
+                points.push_back(ls::sample(sample_idx));
+            }
+        }
+    }
+    else {
+        for (unsigned i = 0; i<nbranch; ++i) {
+            points.push_back(mlocation{i, 0.});
+            for (msize_t sample_idx: util::make_range(cell.morphology().branch_indexes(i))) {
+                points.push_back(ls::sample(sample_idx));
+            }
+        }
+    }
+
+    return std::accumulate(points.begin(), points.end(), ls::nil(), [](auto& l, auto& p) { return join(l, p); });
```

Comment:
Indeed! Missed this one when updating to match locset API changes.

Reply:
Oops, no, I'm wrong. Unlike the other cv policies above, `points` is a vector of `locset`s, rather than a vector of `mlocation`s. (Each locset in the vector is an `ls::sample`.)

Reply:
Ah, true